src/HIE/Bios/Cradle.hs

```diff
@@ -107,6 +109,7 @@ directCradle :: FilePath -> [String] -> Cradle
 directCradle wdir args =
   Cradle {
       cradleRootDir = wdir
+    , cradleDependencies = ["hie.yaml"] -- ^ Flags are supplied via 'hie.yaml'
```

Comment:
I don't think we need to put hie.yaml on the list - that will always go on the list automatically since all of them use it.